flint.ui/src/store/modules/gcbm.js

```diff
@@ -632,7 +633,37 @@ export default {
       }
     }
   },
-  mutations: {},
+  mutations: {
+    save_new_gcbm_job_title(state, title) {
+      console.log('passed title')
+      console.log(title)
+      state.title = title
+      console.log(state.title)
+    }
+  },
 
-  actions: {}
+  actions: {
+    title_setter({ commit }, payload) {
+      commit('save_new_gcbm_job_title', payload)
+      console.log('from title_setter')
+      console.log(this.state.gcbm.title)
+      //dispatch(this.send_new_gcbm_job_title)
```

Comment:
@iamrajiv @waridrox 
Maybe we can refactor this PR to update the simulation title and send it to the API in one click? I wasn't able to achieve that as ` dispatch(this.send_new_gcbm_job_title)` would run without the mutation on line 647 is committed properly, and then returns `undefined`.

Is there any way to do this with `async` & `await`?

Reply:
This might help? https://vuex.vuejs.org/guide/actions.html#composing-actions